server/service/src/sensor/berlinger.rs

```diff
@@ -46,71 +46,42 @@ pub fn get_matching_sensor_log(
     TemperatureLogRepository::new(connection).query_by_filter(filter)
 }
 
-pub fn get_matching_sensor_breach_config(
+fn get_matching_sensor_breach_config(
     connection: &StorageConnection,
-    description: &str,
+    store_id: &str,
+    temperature_breach_config: &temperature_sensor::TemperatureBreachConfig,
     breach_type: &TemperatureBreachRowType,
 ) -> Result<Vec<TemperatureBreachConfig>, RepositoryError> {
     let filter = TemperatureBreachConfigFilter::new()
-        .description(EqualFilter::equal_to(description))
```

Comment:
Matching by description seemed fragile, what if we want to change the format of description, or description is different for configs coming from other sources ?